CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## develop
+
+### Added
+- Added a polyfill for `Array.prototype.find`
```

Comment:
We usually don't repeat the section header in the entry. It should read as one sentence:
```suggestion
- Missing polyfill for `Array.prototype.find`
```

However here it could be treated as a bug-fix so I would go for:
```
### Fixed
- Exception on browsers not supporting `Array.find ` by adding a polyfill
```